### DISCUSSION TITLE
Stricter RouterLink props

### DISCUSSION THREAD

**Discussion by PindaPixel** [posva/unplugin-vue-router#543] [Questions and Help] [answered]
Is it possible to get RouterLink to show an error when it has no params passed, but has at least one required param? E.g.:
```vue
<!-- no error -->
<RouterLink :to="{ name: 'nested/:resource/:id?' }">
    Nested route
</RouterLink>

<!-- Property 'resource' is missing in type '{}' but required... -->
<RouterLink :to="{ name: 'nested/:resource/:id?', params: {} }">
    Nested route
</RouterLink>
```
I had a case where I did the above and introduced a bug that could have been prevented.

**Comment by posva** [comment, marked as answer]
It’s not possible yet, but I want to introduce a flag for it. In the future, I will likely remove the default implicit behavior
